src/matches.rs

```diff
@@ -27,7 +27,8 @@ macro_rules! matches {
 }
 
 /// Work around "error: unexpected token: `an interpolated tt`", whatever that
-/// means.
+/// means. (Probably rust-lang/rust#22819.)
+#[doc(hidden)]
```

Comment:
I wish we could do without stuff like this, since it means you basically have to glob `macro_use` from this crate to get any use, whereas `macro_use(specific, macros)` would probably be better.